API/API.mli

```diff
@@ -373,19 +365,13 @@ sig
   val make_con : ModPath.t -> DirPath.t -> Label.t -> Constant.t
   [@@ocaml.deprecated "alias of API.Names.Constant.make3"]
 
-  val debug_pr_con : Constant.t -> Pp.std_ppcmds
 
-  val debug_pr_mind : MutInd.t -> Pp.std_ppcmds
 
   val pr_con : Constant.t -> Pp.std_ppcmds
 
-  val string_of_con : Constant.t -> string
```

Comment:
Uff. This is an alias of `Constant.to_string`. We can drop this one but I think that, ultimately, it makes sense to keep `Constant.to_string` . `API/PROPERTIES` need to be updated.